tests/test_parse_avro_eager.py

```diff
@@ -26,10 +26,11 @@
 from avro.io import DatumReader, DatumWriter, BinaryDecoder, BinaryEncoder
 from avro.datafile import DataFileReader, DataFileWriter
 from avro.schema import Parse as parse
+from tensorflow_io.core.python.experimental.parse_avro_ops import VarLenFeatureWithRank
```

Comment:
I think `VarLenFeatureWithRank` will need to be exposes the same way as `.make_avro_record_dataset`, so that it is exposed as public API (and you can use it in tests).